client/src/components/credentials/credentialListItem.js

```diff
@@ -1,36 +1,43 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { ListView, Button, Icon, Checkbox } from 'patternfly-react';
+import { SimpleTooltip } from '../simpleTooltIp/simpleTooltip';
 
 class CredentialListItem extends React.Component {
   render() {
     const { item, onItemSelectChange, onEdit, onDelete } = this.props;
 
     let itemIcon;
+    let credentialTypeText;
+
     switch (item.cred_type) {
       case 'vcenter':
         itemIcon = <ListView.Icon type="pf" name="virtual-machine" />;
+        credentialTypeText = 'VCenter';
         break;
       case 'network':
         itemIcon = <ListView.Icon type="pf" name="network" />;
+        credentialTypeText = 'Network';
         break;
       case 'satellite':
         itemIcon = <ListView.Icon type="fa" name="space-shuttle" />;
+        credentialTypeText = 'Satellite';
         break;
       default:
         itemIcon = null;
+        credentialTypeText = '';
```

Comment:
Note: Maybe at some point when we get enough defaults going, maybe we look at populating these variables earlier. In this case around lines 10 & 11, instead of duplicating some effort on a ```default:``` case.

Reply:
Yeah, I hate to set it to '' just to reset it in the valid cases. Likely the default never happens.